### PR TITLE
Weather fetcher code and test with example region

### LINKED ISSUE #8: Weather api test implementation
Should create weather fetcher model and test it.

### PR DESCRIPTION
**Pull request by DonghyunNa**
- Use restler module to make request to api server
- Use xml2json to make json from xml
- Pass "should test"

This changes resolves #8.

### CODE REVIEW

File: models/weather-fetcher.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var rest = require('restler');
+var parser = require('xml2json');
+
+module.exports.fetch = function() {
+
```
Comment (crowjdh) [inline models/weather-fetcher.js]: I think this breakline shouldn't be here :)

File: models/weather-fetcher.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        rest.get('http://www.kma.go.kr/wid/queryDFSRSS.jsp?zone=2723067100')
+        .on('complete', function(data) {
+            if(data instanceof Error){
+                reject(error);
+            } else {
+                resolve(JSON.parse(parser.toJson(data)));
+            }
+        });
+    });
+}
```
Comment (crowjdh) [inline models/weather-fetcher.js]: Missing newline at the end of file

File: models/weather-fetcher.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+var rest = require('restler');
+var parser = require('xml2json');
+
+module.exports.fetch = function() {
+    
+    return new Promise(function(resolve, reject) {
+        var result = {};
```
Comment (crowjdh) [inline models/weather-fetcher.js]: This resule dictionary is redundant and should be removed

File: tests/weather-fetcher.spec.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+describe("Weather fetcher", function() {
+  it('promise return value should exist', function() {
+    return weatherFetcher.fetch().then(function(data){
+        var items = data["rss"]["channel"]["item"]["description"]["body"]["data"];
+        
+        should.exist(data);
+        items.should.have.length(15);
+    });
+  });
+});
```
Comment (crowjdh) [inline tests/weather-fetcher.spec.js]: New line here too :)

File: tests/weather-fetcher.spec.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+const should = require('should');
+const weatherFetcher = require('../models/weather-fetcher.js');
+
+describe("Weather fetcher", function() {
+  it('promise return value should exist', function() {
+    return weatherFetcher.fetch().then(function(data){
+        var items = data["rss"]["channel"]["item"]["description"]["body"]["data"];
+        
+        should.exist(data);
```
Comment (crowjdh) [inline tests/weather-fetcher.spec.js]: This line should be placed before retrieving value from dictionary

File: tests/weather-fetcher.spec.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+const should = require('should');
+const weatherFetcher = require('../models/weather-fetcher.js');
+
+describe("Weather fetcher", function() {
+  it('promise return value should exist', function() {
+    return weatherFetcher.fetch().then(function(data){
+        var items = data["rss"]["channel"]["item"]["description"]["body"]["data"];
```
Comment (crowjdh) [inline tests/weather-fetcher.spec.js]: I'm not sure what is data and items. Can you give more meaningful names to these variables?

File: models/weather-fetcher.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        rest.get('http://www.kma.go.kr/wid/queryDFSRSS.jsp?zone=2723067100')
```
Comment (crowjdh) [inline models/weather-fetcher.js]: I think the zone code(2723067100 in this case) should be passed as a parameter of this funtion.

**Comment by crowjdh** [comment]
Great job! You should be pretty tired by now ;)

### Some suggestions: ###
- Capitalize the first character of the name of PR.
- Add yourself as assignee.(It's under the "Reviewers" section)
- Let's add 성정 as a reviewer.
- What do you think about changing name of the weather fetcher module to "weather-api"? Because
```javascript
weather-fetcher.fetch();
```
seems redundant to me.
- Add line below to your PR comment to close issue that you're assigned
```
This changes resolves #8.
```

### Questions ###
- I'm not sure about what your comment "Pass should test" exactly means. You mean "Should pass test"?

**Review by crowjdh — CHANGES_REQUESTED** [review]
I specified some issues to resolve. Please take your time to resolve :)

File: models/weather-fetcher.js (hunk 7 above)
Comment (DonghyunNa) [reply models/weather-fetcher.js]: 지금은 기본값만 넣어두고 차후에 parameter 받아서 지역별로 데이터 받아와야 합니다.

**Review by crowjdh — APPROVED** [review]

**Review by yeareater — APPROVED** [review]

**Pull request merged**